Source/WTF/wtf/Assertions.h

```diff
@@ -681,6 +681,12 @@ constexpr bool assertionFailureDueToUnreachableCode = false;
         WTFLogAlways("%s", stream.release().utf8().data()); \
     } while (0)
 
+#define WTF_ALWAYS_LOG(commands) do { \
```

Comment:
It would be good to replace the `ALWAYS_LOG_WITH_STREAM` macro, so we don't have two competing implementations.

There are only a handful uses of `ALWAYS_LOG_WITH_STREAM` when performing a code search.

Reply:
I suggest to let it there a bit for a transition period, and we can remove in a future patch ?